defaults/main.yml

```diff
@@ -22,9 +22,9 @@ promtail_loki_server_url: http://127.0.0.1:3100
 promtail_config_clients:
   - url: "{{ promtail_loki_server_url }}/loki/api/v1/push"
 
-promtail_config_positions: {}
-#  promtail_config_positions:
-#    filename: /tmp/positions.yaml
+promtail_positions_path: /var/lib/promtail/positions.yml
+promtail_config_positions:
+  filename: "{{ promtail_positions_path }}"
```

Comment:
Why the need for adding another variable? We could set the default value directly to
```
promtail_config_positions:	promtail_config_positions:
  filename: /var/lib/promtail/positions.yml
```

Reply:
convenience, actually. for most customizations the proposed `promtail_positions_path` should be enough. the `promtail_config_positions` still allows to customize the whole config section.